features/_your_details/age_validation.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Display validation messages for age
+  As a user of the Workplace Pension Contribution Calculater tool
+  In order to get a clearer idea on how to progress through the app
+  I want to receive helpful messages that clearly explains where I may be going wrong
+
+  """ Background
+    Add server-side validation messages to page 1 for user-selections that should not allow the user to progress without amending or checking their inputs.
+    These messages should be placed underneath the affected input. and be highlighted by being red in colour.
+  """
```

Comment:
Do we need this background info? Also there is an extraneous full stop after input.